src/utils/reqwest_resume/tests/get_test.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+use crate::utils::reqwest_resume::get;
+use bytes::Bytes;
+use futures::StreamExt;
+use http_range_header::parse_range_header;
+use hyper::header::{self, HeaderValue};
+use hyper::service::{make_service_fn, service_fn};
+use hyper::{Body, Request, Response, Server};
+use std::convert::Infallible;
+use std::time::Duration;
+use tokio::time::sleep;
+
+const BUFFER_LEN: usize = 4096 * 2;
+const CHUNK_LEN: usize = 4096;
```

Comment:
Could you let me know why you picked these values? What are the constraints on sensible values here?

Reply:
Those were arbitrarily chosen. We could use something smaller or bigger here.
The only constraint is that `CHUNK_LEN < BUFFER_LEN.`

However, I'm not sure if we can assert on `BUFFER_LEN` in case of non-resumable get test:

`assert_eq!(item.unwrap().len(), CHUNK_LEN);`

Testing this with a chunk value `10000` shows that we get only `8080` in `test_resumable_get.`
A better approach would be to test the actual data transmitted.